yorkie/proto/src/main/proto/yorkie/v1/resources.proto

```diff
@@ -47,7 +47,7 @@ message Change {
 }
 
 message ChangeID {
-  uint32 client_seq = 1;
+  uint64 client_seq = 1;
```

Comment:
As stated in the PR description above, I changed `uint32` to `uint64` since Java does not support `uint32` and automatically converts it to `int32`.  
(If `minSyncedTicket` exceeds `Int.MAX_VALUE`, it is converted incorrectly, and GC does not work properly.)

If changing it can have potential issues, I’ll find another solution for it.
(cc. @hackerwins)

Reply:
This was discussed [here](https://github.com/yorkie-team/yorkie-android-sdk/pull/12#discussion_r988468911).
Since exceeding `Int.MAX_VALUE` is unlikely to happen in real world, I think it's okay to stick with it.
Also I'm not sure having different protobuf models between different platforms is a good idea.
But this is something that should be decided by @hackerwins.

Reply:
It appears that the `minSyncedTicket.delimiter` is sometimes set to the maximum value of uint32 for GC. In the Android SDK, this value is then converted to -1, resulting in nothing being garbage collected.
If this is the only case that exceeding `Int.MAX_VALUE`, we can just consider -1 as `MAX_DELIMITER` without changing protobuf models:
```kotlin
internal fun PBTimeTicket.toTimeTicket(): TimeTicket {
    return TimeTicket(
        lamport = lamport,
        delimiter = if (delimiter == -1) MAX_DELIMITER else delimiter,
        actorID = actorId.toActorID(),
    )
}
```
I think it needs to be checked by @hackerwins.

Reply:
I agree it would be better to use `UInt` for kotlin models without changing protobuf models.
done in [aad775c](https://github.com/yorkie-team/yorkie-android-sdk/pull/123/commits/aad775cd93e1fa6e14d6f25484bfc098f4015446).